data/functions/itemized.sql

```diff
@@ -15,3 +121,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
```

Comment:
I would recommend _naming_ the indexes and constraints so that _renaming_ is not required - after all these are ephemeral names.